packages/commonwealth/server/routes/generateImage.ts

```diff
@@ -15,13 +15,19 @@ let openai: OpenAI | undefined = undefined;
 
 const log = logger(import.meta);
 
-try {
-  openai = new OpenAI({
-    organization: 'org-D0ty00TJDApqHYlrn1gge2Ql',
-    apiKey: process.env.OPENAI_API_KEY,
-  });
-} catch (e) {
-  log.error('OpenAI initialization failed.', e);
+if (process.env.OPENAI_API_KEY) {
+  try {
+    openai = new OpenAI({
+      organization: 'org-D0ty00TJDApqHYlrn1gge2Ql',
```

Comment:
shouldn't we require external devs to use their own organization?

Reply:
good catch, fixed